frontend/.env.sample

```diff
@@ -1,6 +1,7 @@
 PORT=3000
 NODE_ENV=development
 API_V3_URL=//trase.earth
+UNIT_LAYERS_API_URL=//trase.earth
 GOOGLE_ANALYTICS_KEY=UA-87285243-1
```

Comment:
is this a leak or is it fine?

Reply:
It should be fine. It will be the default for production